templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -687,6 +687,7 @@ Resources:
           - Effect: Allow
             Action:
               - 'ssm:PutParameter'
+              - 'ssm:DeleteParameter'
```

Comment:
Was it possible to delete the stack with the ansible pinned manually before? My question is, are we doing this just for the reaper or was there customer impact as well? Is the other option to extend Lambda permissions?